src/ducks/GainersAndLosers/GainersLosersPage.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react'
+import { graphql } from 'react-apollo'
+import { Panel } from '@santiment-network/ui'
+import GainersLosersTable from './GainersAndLosersTable'
+import { Tabs } from '@santiment-network/ui'
+import { allProjectsGQL } from '../../pages/Projects/allProjectsGQL'
+import styles from './GainersAndLosersPage.module.scss'
+import { mapItemsToKeys } from '../../utils/utils'
+
+const statusMapping = {
+  Gainers: 'GAINER',
+  Losers: 'LOSER'
+}
+
+const timeWindowMapping = {
+  '24h': '2d',
+  '2w': '15d'
+}
+
+class GainersAndLosersPage extends Component {
+  state = {
+    timeWindow: '2d',
+    status: 'GAINER'
+  }
+
+  render () {
+    const { timeWindow, status } = this.state
+    const { allProjects } = this.props
+    return (
+      <div className='page'>
+        <h1>Top Gainers And Top Losers</h1>
+        <Panel className={styles.table}>
+          <div className={styles.tabs}>
+            <Tabs
+              options={['Gainers', 'Losers']}
+              defaultSelectedIndex='Gainers'
+              onSelect={selectedTab =>
```

Comment:
**[D-1-1]** Should be a class instance method, e.g. `onTabSelect`, otherwise why to create dynamically allocated closures in the class component